LightBlue/Hosts/WebHost.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.IO;
+using Serilog;
+using System.Configuration;
+
+namespace LightBlue.Hosts
+{
+    public class WebHost
+    {
+        private readonly Settings _settings;
+        private WebHostProcess _iis;
+        public WebHost(Settings settings)
+        {
+            _settings = settings;
+        }
+
+        public void Start()
+        {
+            Log.Logger = new LoggerConfiguration()
```

Comment:
Logging only occurs during setup so I can see what the data directory is and what assembly it is executing